tests/conftest.py

```diff
@@ -1763,3 +1764,9 @@ def disable_version_and_agent_cleanup_job():
     orchestrationservice.PERFORM_CLEANUP = False
     yield
     orchestrationservice.PERFORM_CLEANUP = old_perform_cleanup
+
+
+@pytest.fixture(scope="function")
+async def cleanup_log_instance():
+    InmantaLoggerConfig.clean_instance()
+    yield
```

Comment:
Shouldn't we call this from the `reset_all_objects()` method instead? Then it runs before and after each test case. I don't think it's a good idea to leave singleton in a dirty state across runs of test cases.

Reply:
It is better, I'll change it :)